otherlibs/win32unix/unix.ml

```diff
@@ -393,6 +393,12 @@ let mkfifo _name _perm = invalid_arg "Unix.mkfifo not implemented"
 external readlink : string -> string = "unix_readlink"
 external symlink_stub : bool -> string -> string -> unit = "unix_symlink"
 
+let normalize_path path =
```

Comment:
Given that it's not path normalisation, perhaps this could be `normalize_slashes` or `normalize_separator`?